tests/test_draft_resource.py

```diff
@@ -79,4 +80,57 @@ def test_create_draft_of_existing_record(app, client, record_service,
     fields_to_check = ['pid', 'metadata', 'revision',
                        'created', 'updated', 'links']
 
+    for field in fields_to_check:
+        assert field in response_fields
+
     assert response.json['metadata']['title'] == orig_title
+
+
+def test_publish_draft_of_new_record(app, client, draft_service, input_record,
+                                     fake_identity):
+    """Test draft publication of a non-existing record.
+
+    It has to first create said draft.
+    """
+    # Needs `app` context because of invenio_access/permissions.py#166
+    # Crate the draft
```

Comment:
```suggestion
    # Create the draft
```

Reply:
I just wanted a crate of drafts ... 😂 Thanks for the catch :)